lib/iex/lib/iex/introspection.ex

```diff
@@ -313,6 +315,21 @@ defmodule IEx.Introspection do
     dont_display_result
   end
 
+  defp print_type_doc(module, type) do
+    case get_type_doc(module, type) do
+      {_, nil} -> nil
+      {_, description} -> print_doc(inspect(module), description)
```

Comment:
We don't need to print the module name, let's just skip the header and print the type followed by the docs!